lib/components/artist/related_artists/related_artist.js

```diff
@@ -49,7 +52,7 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
-  initialVariables: { width: 150, height: 100 },
+  initialVariables: { imageWidth: ~~(imageWidth), imageHeight: ~~(imageWidth / 1.5) },
```

Comment:
lol is it too much? I figured it was prettier than using `Math`: http://stackoverflow.com/questions/596467/how-do-i-convert-a-float-number-to-a-whole-number-in-javascript